src/Core/Domain/Enum/Setting.php

```diff
@@ -29,4 +29,59 @@ final class Setting
      * @var string
      */
     public const SETTING_TYPE_SDK = 'sdk';
+
+    /**
```

Comment:
Please add a short description as it was done for the constants above. The same for the rest.